sqlglot/expressions.py

```diff
@@ -1132,7 +1185,7 @@ class Add(Binary):
     pass
 
 
-class And(Binary):
+class And(Binary, Condition):
```

Comment:
should
In and Like also be conditions?

Reply:
Oof, yes. I did another pass through to try and tag everything that should be a condition.